### PR TITLE
add footer to android info asset

### PR DESCRIPTION
**Pull request by mashrurc-alt**
This PR addresses https://github.com/player-ui/player/issues/163

### CODE REVIEW

File: plugins/reference-assets/android/src/main/res/layout/info.xml [hunk 1]
```diff
@@ -33,4 +33,15 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/info_primary_info" />
 
+    <LinearLayout
```
Comment (brocollie08) [inline plugins/reference-assets/android/src/main/res/layout/info.xml:36]: is there a reason for the choice of `LinearLayout` here? It looks like we only handle `footer` as a singular asset

**Comment by brocollie08** [comment]
Could you please add to the tests for this asset

**Comment by sugarmanz** [comment]
Closing, was done in #398

**Pull request closed without merging**